ui/pages/bridge/utils/quote.ts

```diff
@@ -33,27 +45,169 @@ export const isValidQuoteRequest = (
   );
 };
 
-export const getQuoteDisplayData = (quoteResponse?: QuoteResponse) => {
-  const { quote, estimatedProcessingTimeInSeconds } = quoteResponse ?? {};
-  if (!quoteResponse || !quote || !estimatedProcessingTimeInSeconds) {
-    return {};
-  }
+export const calcToAmount = (
+  { destTokenAmount, destAsset }: Quote,
+  exchangeRate: number | null,
+) => {
+  const normalizedDestAmount = calcTokenAmount(
+    destTokenAmount,
+    destAsset.decimals,
+  );
+  return {
+    raw: normalizedDestAmount,
```

Comment:
Hmm yea this looks like a decimal amount here, but usually I see raw meaning the atomic amount.